mc/gui/general_settings_wt.py

```diff
@@ -0,0 +1,39 @@
+import mc.mc_global
+import mc.model
+import os
+from PyQt5 import QtWidgets
+from shutil import copyfile
+from sys import platform
```

Comment:
We can also get the platform from Qt, right now we have "kernel type: linux". I'm not sure though which of these options (getting the platform from Qt or from Python) is safest